activitysim/examples/example_mtc_extended/configs/vehicle_type_choice_op2_fuel_type_probs.csv

```diff
@@ -0,0 +1,101 @@
+vehicle_type,Gas,Diesel,Hybrid,PEV,BEV
+Car_1,.9205,0.0020,0.0468,0.0153,0.0155
+Car_2,.9095,0.0120,0.0546,0.0082,0.0157
+Car_3,.9038,0.0163,0.0595,0.0115,0.0088
+Car_4,.8951,0.0167,0.0684,0.0107,0.0090
+Car_5,.9079,0.0155,0.0646,0.0065,0.0054
+Car_6,.9365,0.0137,0.0436,0.0019,0.0043
+Car_7,.9131,0.0117,0.0744,0.0006,0.0002
+Car_8,.9541,0.0096,0.0355,0.0007,0.0000
+Car_9,.9548,0.0037,0.0409,0.0004,0.0001
+Car_10,0.9530,0.0015,0.0451,0.0003,0.0001
+Car_11,0.9676,0.0096,0.0225,0.0003,0.0000
+Car_12,0.9630,0.0059,0.0306,0.0003,0.0002
+Car_13,0.9773,0.0061,0.0164,0.0000,0.0002
+Car_14,0.9808,0.0107,0.0082,0.0002,0.0000
+Car_15,0.9844,0.0088,0.0065,0.0000,0.0003
+Car_16,0.9894,0.0056,0.0050,0.0000,0.0000
+Car_17,0.9947,0.0031,0.0022,0.0000,0.0000
+Car_18,0.9934,0.0066,0.0000,0.0000,0.0000
+Car_19,0.9940,0.0050,0.0000,0.0000,0.0010
+Car_20,0.9877,0.0120,0.0002,0.0000,0.0002
+Van_1,0.9728,0.0163,0.0072,0.0036,0.0000
+Van_2,0.9927,0.0073,0.0000,0.0000,0.0000
+Van_3,0.9873,0.0089,0.0025,0.0000,0.0013
```

Comment:
...like here...why would a van that is 3 years old have a higher probability of being a BEV than 2 years old? If these #s are this "wobbly" then we should smooth them out to reduce implied meaningful variation.

Reply:
See above comment.